content/docs/en/elements/components/side-drawer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: SideDrawer
+apiRef: https://docs.telerik.com/devtools/nativescript-ui/Controls/NativeScript/SideDrawer/overview
```

Comment:
Probably link should be: https://docs.telerik.com/devtools/nativescript-ui/Controls/Vue/SideDrawer/overview or the actual API reference at http://docs.telerik.com/devtools/nativescript-ui/api/classes/radsidedrawer.html